src/VecSim/algorithms/brute_force/brute_force_multi.h

```diff
@@ -57,12 +58,11 @@ class BruteForceIndex_Multi : public BruteForceIndex<DataType, DistType> {
 template <typename DataType, typename DistType>
 int BruteForceIndex_Multi<DataType, DistType>::addVector(const void *vector_data, labelType label) {
 
-    DataType normalized_data[this->dim]; // This will be use only if metric == VecSimMetric_Cosine
+    DataType normalized_blob[this->dim]; // This will be use only if metric == VecSimMetric_Cosine.
```

Comment:
why the rename?

Reply:
it was either normalized_blob or normalized_data so I chose one and made it consistent across the library